### ISSUE TITLE
two typos in twisted.web.template docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4973]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4973|
|Type|defect|
|Created|2011-03-21 19:04:42Z|
|Branch|https://github.com/twisted/twisted/tree/template-typos-4973|

  * The module docstring says _An convenience_
  * The `flattenString` docstring says _into a single sring_

<details><summary>Searchable metadata</summary>

```
trac-id__4973 4973
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_template_typos_4973 branches/template-typos-4973
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1300734282000000 1300734282000000
changetime__1300736715000000 1300736715000000
version__None None
owner__exarkun exarkun
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [31362]) Branching to 'template-typos-4973'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [31363]) Trivial docstring fixes

refs [#4973](https://github.com/twisted/twisted/issues/4973)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set owner to @exarkun|
|-|-|

Trivial indeed.  Merge please.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [31365]) Merge template-typos-4973

Author: exarkun
Reviewer: glyph
Fixes: [#4973](https://github.com/twisted/twisted/issues/4973)

Fix two trivial typos in the `twisted.web.template` API docs.
